backend/dal/Migrations/Initial/PostDeploy/01-ProjectStatus.sql

```diff
@@ -51,49 +51,57 @@ INSERT INTO dbo.[ProjectStatus] (
 ), (
     5
     , 5
+    , 'Approval Confirmation'
+    , 0
+    , 'Confirm approval has been received from the right approval authority.'
+    , '/project/approval/confirm'
+    , 'ReviewDisposal'
+), (
+    6
+    , 6
     , 'Submitted'
     , 0
     , 'The project has been submitted for review to be added to the Surplus Property Program.'
     , '/project/submitted'
     , 'SubmitDisposal,ReviewDisposal'
 ), (
-    6
-    , 6
+    7
+    , 7
     , 'Property Review'
     , 0
     , 'Property information review.'
     , '/project/property/review'
     , 'ReviewDisposal'
 ), (
-    7
-    , 7
+    8
+    , 8
     , 'Document Review'
     , 0
     , 'Documentation reviewed (Surplus Declaration & Readiness Checklist, Triple Bottom Line).'
     , '/project/documentation'
     , 'ReviewDisposal'
 ), (
-    8
-    , 8
+    9
+    , 9
     , 'First Nation Consultation'
     , 0
     , 'First Nation Consultation process.'
     , '/project/fist/nation/consultation'
     , 'ReviewDisposal'
 ), (
-    9
-    , 9
+    10
+    , 10
     , 'Approved'
     , 0
     , 'The project has been approved to be added to the Surplus Property Program.'
     , '/project/approved'
     , 'ReviewDisposal'
 ), (
-    10
-    , 10
+    11
+    , 11
     , 'Denied'
     , 0
     , 'The project has been denied to be added to the Surplus Property Program.'
     , '/project/denied'
-    , 'ReviewDisposal'
+    , 'SubmitDisposal'
```

Comment:
This is also incorrect at this point in time.  The template does not provide a **Denied** state on the **Start a Project**